### ISSUE TITLE
New dependency on "elm-pep" creates unexpected warning 

### ISSUE THREAD

**Issue by awendt** [openlayers/openlayers#10706]
**Describe the bug**

In a project of mine, after upgrading from 6.1.1 to 6.2.1, I get the following warning from Parcel:

```
Could not load source file "../src/elm-pep.js" in source map of "../node_modules/elm-pep/dist/elm-pep.js"
```

The diff is here: https://github.com/awendt/familienradwege-website/pull/31. According to comment, the warning appears because `elm-pep` is not packaged correctly: https://github.com/parcel-bundler/parcel/issues/2356#issuecomment-444814230

**To Reproduce**
1. Copy an example from the website e.g. https://openlayers.org/en/latest/examples/drag-rotate-and-zoom.html
2. Run `npm install && npm run start`
3. See warning

**Expected behavior**

I expect the warning "Could not load source file …" not to appear, just like it does when using 6.1.1

**Comment by ahocevar** [comment]
I just published v1.0.6 of elm-pep, which includes the `src/` folder. So upgrading to that version should fix your issue.

**Comment by awendt** [comment]
Works for me – thanks for the prompt reply and sorry it took _me_ so long to get back to you! ⌛️
